plugins/entando-plugin-jacms/src/main/java/com/agiletec/plugins/jacms/apsadmin/resource/MultipleResourceAction.java

```diff
@@ -103,6 +103,7 @@ public void validate() {
 
     @Override
     public String edit() {
+        logger.debug("Edit");
```

Comment:
This probably needs some context. "Edit in multiple resource action for id {} ...." or something like that